app/src/main/res/layout/list_stream_grid_item.xml

```diff
@@ -21,6 +21,32 @@
         android:src="@drawable/dummy_thumbnail"
         tools:ignore="RtlHardcoded"/>
 
+    <TextView
+        android:id="@+id/itemInHistoryIndicatorView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignStart="@id/itemThumbnailView"
+        android:layout_alignTop="@id/itemThumbnailView"
+        android:layout_alignEnd="@id/itemThumbnailView"
+        android:layout_alignBottom="@id/itemThumbnailView"
+        android:layout_gravity="top|left"
+        android:background="@drawable/item_in_history_indicator_background"
+        android:ellipsize="end"
```

Comment:
Is this a good idea? In theory the ellipsize should never be triggered, but even if it is I'd prefer marquee
(also below)

Reply:
I think if it was just one view or even a list that you can select an element it'd be fine with the marquee, but with this much items, I don't think so. What about the users that use the bigger fonts and have a bunch of items that are already played, wouldn't that be too distracting?

I think that's why the marquee effect is designed to only start when the item is actually selected (otherwise you have to call `setSelected(true)` manually).

Maybe an auto-sizing text view would work better here, even though we won't be able adapt all sizes, most of them will be fine with that. It'd also work better on the mini version of the stream item.

Reply:
What about using marquee, but never calling `setSelected(true)` on it, so that the `IN HISTORY` text fades at the end instead of showing three points? Anyway, if this would look too strange, I'm ok with not changing anything, since "in theory the ellipsize should never be triggered"